conf_files/huntsman.yaml

```diff
@@ -51,12 +51,16 @@ mount:
     driver: bisque
     model: 45
     template_dir: resources/bisque
-filterwheel:
-  flat_field_order:
-    - one
-    - deux
-    - drei
-    - quattro
+observatory:
+  flat_fields:
+    flat_field_order:
+      - one
+      - deux
+      - drei
+      - quattro
+    max_num_exposures: 5
+    max_attempts: 10
+    max_exptime: 120
```

Comment:
It's a little weird to have this under the `observatory` key since the entire config is related to the observatory.  Can we just leave it at `flat_fields`?